core/src/main/resources/reference.conf

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+// Required until https://github.com/akka/akka/pull/28333 is available
+akka.actor.allow-java-serialization = on
```

Comment:
Shouldn't each application decide for itself whether to enable this or not?
Do we use java serialization in the tests? If so, then it might make more sense to enable this in the test config

Reply:
Ah, you're right, it should just be for our tests.